.github/workflows/run_tests_postgresql.yml

```diff
@@ -0,0 +1,46 @@
+name: build-postgres
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 21
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest]
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: cornflow-test
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgresadmin
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -U -r requirements-dev.txt
+    - name: Run unit tests
+      run: python -m unittest discover -s cornflow.tests.unit
+      env:
+        FLASK_ENV: testing
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: 5432
```

Comment:
should these two be here? or in the postgres service configuration above?

Reply:
In the examples given by GitHub they place those values there:

https://docs.github.com/en/actions/guides/creating-postgresql-service-containers#running-jobs-directly-on-the-runner-machine

Reply:
I checked the example: actually we do not need those env vars.

Check what they say in the comments:
```
# Runs a script that creates a PostgreSQL table, populates
# the table with data, and then retrieves the data
run: node client.js
# Environment variables used by the `client.js` script to create
# a new PostgreSQL table.
env:
  # The hostname used to communicate with the PostgreSQL service container
  POSTGRES_HOST: localhost
	  # The default PostgreSQL port
	  POSTGRES_PORT: 5432
```
So the environment variables are for the node client.js example.

Reply:
I have deleted the environment variables and it works. I will perform the merge.